### ISSUE TITLE
Wrong doc for File Permissions in Symfony 3.4

### ISSUE THREAD

**Issue by lukepass** [symfony/symfony-docs#9697]
Hello, I noticed that the page "Setting up or Fixing File Permissions" for Symfony 3.4 is about Symfony 4:
https://symfony.com/doc/3.4/setup/file_permissions.html

It should have documentation similar to this:
https://symfony.com/doc/3.3/setup/file_permissions.html

I was looking for this script:
```
HTTPDUSER=$(ps axo user,comm | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1)

 sudo setfacl -dR -m u:"$HTTPDUSER":rwX -m u:$(whoami):rwX var
 sudo setfacl -R -m u:"$HTTPDUSER":rwX -m u:$(whoami):rwX var
```

**Comment by javiereguiluz** [comment]
That's correct ... but this is probably intended. The reason is that Symfony 3.4 docs assume that the reader is using Symfony Flex and the Symfony Skeleton to create and manage projects. That's why the permission problem is no longer a problem and everything works without the developer having to do anything.

**Comment by lukepass** [comment]
Uhm I don't think so because in the setup docs the symfony installer is used:

https://symfony.com/doc/3.4/setup.html

```
symfony new my_project_name 3.4
```

Flex is mentioned in another article and it's not required.

**Comment by javiereguiluz** [comment]
@lukepass I'm sorry but I was wrong. You are right that in 3.4 we still use the "old Symfony". However, regarding the issue reported, I think it's correct:

* 3.3: https://symfony.com/doc/3.3/setup/file_permissions.html long article, explains 4 methods and uses things like `app.php` and `app_dev.php`
* 3.4: https://symfony.com/doc/3.4/setup/file_permissions.html long article, explains 4 methods and uses things like `app.php` and `app_dev.php`
* 4.0: https://symfony.com/doc/4.0/setup/file_permissions.html short article, explains just 1 method and uses only `index.php`

In fact, I can see in the 3.4 version the script you said you were missing:

![doc_error](https://user-images.githubusercontent.com/73419/40435201-6e9c7806-5eb0-11e8-9e9e-0297bb8cc717.png)

So, I'm closing because this was probably a temporary error and now it looks fixed. If you disagree, please reopen. Thanks!
